oauth2client/contrib/gce.py

```diff
@@ -45,6 +48,31 @@
 """
 
 
+def _get_service_account_email(http_request=None):
+    """Get the GCE service account email from the current environment.
+
+    Args:
+        http_request: callable, (Optional) a callable that matches the method
+                      signature of httplib2.Http.request, used to make
+                      the request to the metadata service.
+
+    Returns:
+        string, The service account email found.
+
+    Raises:
+        RuntimeError, If the request to the metadata service fails.
+    """
+    if http_request is None:
+        http_request = httplib2.Http().request
+    response, content = http_request(
+        _DEFAULT_EMAIL_METADATA, headers={'Metadata-Flavor': 'Google'})
+    if response.status == http_client.OK:
+        content = _from_bytes(content)
+        return content
+    else:
+        raise RuntimeError(response, content)
```

Comment:
@nathanielmanistaatgoogle I'm sorry I didn't point this out sooner. I don't think `RuntimeError` is appropriate here, but I didn't want to implement a do-nothing

```python
class SneezingError(Exception): pass
```

as is the tradition in `oauth2client`. WDYT?